### DISCUSSION TITLE
How is the MR create user determined?

### DISCUSSION THREAD

**Discussion by andorfermichael** [renovatebot/renovate#10648] [Request Help] [answered]
I am interested in how the user (account) for MR creation is determined.

Currently, I face the problem that all MRs created by the renovate-runner are created with my own account.

Is this because I have created the scheduled pipeline? However, I have already transferred ownership to another user account.
Is it because it uses an access token of my personal account?
Is there an option to define the user (account) to be used for MR creation?

Example:
![Screenshot from 2021-06-29 12-17-36](https://user-images.githubusercontent.com/11174651/123781439-891d5780-d8d4-11eb-91f5-0c8c39b0f35a.png)

**Comment by rarkins** [comment]
How *exactly* are you running Renovate and what do you mean by "I have already transferred ownership to another user account."?

**Comment by viceice** [comment, marked as answer]
The `RENOVATE_TOKEN` is used to determine the user account, so you need to use another account for creating the `PAT` or create a project access token (selfhosted gitlab or premium account required) .

**Reply by andorfermichael** [reply]
project access token did not work but PAT with another account did.
